src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Excavator started its work.");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Excavator stopped its work.");

```

Comment:
Similarly, the message 'Excavator stopped its work.' could be more descriptive. Indicate what kind of work the Excavator has stopped doing.